### PR TITLE
games-action/minetest_game: version bump to 0.4.16

### PR DESCRIPTION
**Pull request by Anth0rx**
Gentoo-Bug: 608466

### CODE REVIEW

File: games-action/minetest_game/Manifest [hunk 1]
```diff
@@ -1,2 +1,7 @@
...
+EBUILD minetest_game-0.4.13.ebuild 589 SHA256 7b52e0d19928a05b94c9924037441c93d8860d7711766503e09c78b380bee013 SHA512 666d33e55042c9253402cefc8ad4b286c33b97292e7620edf436445533abc9ad61550b6ae8252942ced3360c340d19f3304ad5072c93d1165fed3f0ad6169214 WHIRLPOOL bd26e1dc9510ee9ea99f4bbefa0fd49a3aec292278003cd2b435b2c9d8b89943a55332b3f24d8f2541130f09611acc206c29e745a16d7ba1878c51e53b9b5d6b
+EBUILD minetest_game-0.4.14.ebuild 581 SHA256 4281c64c1a0b0f17534fd534fd72af8722406a0190a50107b6ba0304a2ec1dfa SHA512 8e953175f5dfd5e99de7223ba754b63d6259528681ffdea018118002f29b9e22c5508598df168ea74a7fe618ffc4e0cb2bb2bf8655169ce97775f0a5cc7607ac WHIRLPOOL 8c13e645121b5f95e8075ecd80300fe190da42b16afe14ebafa4aa7962bc5239afdc84b7ec79258111e880401f6cc9f36d818ccf2f16cc74ea422161762c9418
+EBUILD minetest_game-0.4.15.ebuild 581 SHA256 4281c64c1a0b0f17534fd534fd72af8722406a0190a50107b6ba0304a2ec1dfa SHA512 8e953175f5dfd5e99de7223ba754b63d6259528681ffdea018118002f29b9e22c5508598df168ea74a7fe618ffc4e0cb2bb2bf8655169ce97775f0a5cc7607ac WHIRLPOOL 8c13e645121b5f95e8075ecd80300fe190da42b16afe14ebafa4aa7962bc5239afdc84b7ec79258111e880401f6cc9f36d818ccf2f16cc74ea422161762c9418
```
Comment (SoapGentoo) [inline games-action/minetest_game/Manifest]: this is a think manifest. You can either use

1. `repoman` + GPG (recommended, takes a bit of effort to setup)
2. `ebuild` manifest

to get rid of the EBUILD lines. The CI is failing due to these lines

File: games-action/minetest_game/minetest_game-0.4.15.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+LICENSE="GPL-2 CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=games-action/minetest-${PV}"
+
+src_install() {
+	insinto /usr/share/minetest/games/${PN}
```
Comment (SoapGentoo) [inline games-action/minetest_game/minetest_game-0.4.15.ebuild]: have you tested whether you can drop the `/games` part? Nowadays, games are just normal "applications"

Comment (Anth0rx) [inline games-action/minetest_game/minetest_game-0.4.15.ebuild]: The ```/usr/share/minetest/games/``` is for the "default" minetest subgames (mod collections actively maintained to work with each other). In this case the "default" ```minetest_game``` and the ```minimal``` development game. User added subgames normally reside inside ```~/.minetest/games/```.

Comment (quazgar) [inline games-action/minetest_game/minetest_game-0.4.15.ebuild]: The `games` subfolder looks OK to me. But I believe that adding another `${PREFIX}` here won't hurt.

**Comment by quazgar** [comment]
Tested myself and works for me.

File: games-action/minetest_game/minetest_game-0.4.15.ebuild (hunk 2 above)
Comment (SoapGentoo) [inline games-action/minetest_game/minetest_game-0.4.15.ebuild]: nah, insinto is a Gentoo command, and is prefix aware, not need to add `${EPREFIX}`

**Comment by Anth0rx** [comment]
How exactly should I do the rebase?

**Comment by Anth0rx** [comment]
I tested the new minetest_game version and it works fine for me.

**Comment by soredake** [comment]
`git pull upstream master -r`

**Comment by Anth0rx** [comment]
The QA check failures are due to the unavailability of the `games-action/minetest-0.4.16` package in the upstream gentoo ebuild repository. So first the #3874 PR needs to be merged.

File: games-action/minetest_game/minetest_game-0.4.16.ebuild [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+LICENSE="GPL-2 CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=games-action/minetest-${PV}"
+
+src_install() {
+	insinto ${EPREFIX}/usr/share/minetest/games/${PN}
```
Comment (mgorny) [inline games-action/minetest_game/minetest_game-0.4.16.ebuild]: `insinto` includes EPREFIX already, so don't duplicate it.

**Comment by Anth0rx** [comment]
As I merged the commits into the `minetest` branch (PR #3874) I think this PR can be closed.

**Comment by mgorny** [comment]
Ok, thanks. I'll get to that in a few minutes.

**Pull request closed without merging**